Rename Recipe DSL to Chef Infra Language (#2857)
chef/chef-web-docs: content/infra_language.md

+++
title = "About the Chef Infra Language"
draft = false

gh_repo = "chef-web-docs"

aliases = ["/dsl_recipe.html", "/dsl_recipe"]

[menu]
  [menu.infra]
    title = "Language Overview"
    identifier = "chef_infra/cookbook_reference/infra_language/infra_language.md Language Overview"
    parent = "chef_infra/cookbook_reference/infra_language"
    weight = 10
+++

{{% infra_lang_summary %}}

> The Chef Infra Language is based on Ruby, allowing you to utilize the power of Ruby when the built-in language doesn't meet your needs out of the box. If you'd like to learn more about extending your Chef Infra code by using Ruby see our [Ruby Guide](/ruby/) for further information on Ruby functionality.

Reviewer: Utilize -> use

Reviewer: ```suggestion
The Chef Infra Language is based on Ruby, giving you access to the power of Ruby when the built-in language doesn't meet your needs. Learn more about extending your Chef Infra code in our [Ruby Guide](/ruby/).
```

## Resources

> Resources are the cornerstone of the Chef Infra Language. Resources define the desired state of an object on a system. A resource can be as simple as a directory or as complex or a complete security policy. Chef Infra Client ships with over 150 resources for configuring components such as packages, file, directories, or firewalls. For more information on resources in Chef Infra Client including a complete list of those included out of the box see [Resources](/resources).

Reviewer: Can you give me something more specific term (or set of terms) than "object" in the "desired state of an object on a system" 

```suggestion
The Chef Infra Language expresses the desired state of configurable objects in your system in resources. They are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.
```

{{% infra_lang_summary %}}

The Chef Infra Language is based on Ruby, giving you access to the power of Ruby when the built-in language doesn't meet your needs. Learn more about extending your Chef Infra code in our [Ruby Guide](/ruby/).

## Resources

> The Chef Infra Language expresses the desired state of configurable objects in your system in resources. They are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.

Reviewer: ```suggestion
The Chef Infra Language expresses the desired state of configurable objects in your system in resources. Resources are the building blocks of recipes and they are central to understanding configuration management with Chef Infra. Some resources are as simple as a directory name, while others as complex as a complete security policy. Chef Infra Client ships with over 150 pre-packaged resources for configuring components such as packages, files, directories, and firewalls. Learn more in the Chef Infra Client [Resources](/resources) documentation.
```

Reviewer: Unclear antecedent
